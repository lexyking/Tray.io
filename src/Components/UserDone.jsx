import React from 'react';
import imgSrc from '../images/user-check-solid.svg'

function userDone() {
  return (
    <section className="mainSection">
      <section className="formSection">
        <section className="userImg">
          <img src={imgSrc} alt='' />
        </section>

        <p>Please verify your email address, you should have received an email
          from us already
        </p>
      </section>
    </section>
  )
}

export default userDone
