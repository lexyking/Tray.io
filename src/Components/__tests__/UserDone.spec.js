import React from 'react';
import { create } from 'react-test-renderer';
import UserDone from '../UserDone';


describe('UserDone component', () => {
  test('Render Correctly', () => {
    const userDone = create(<UserDone />);
    expect(userDone.toJSON()).toMatchSnapshot();
  })
})