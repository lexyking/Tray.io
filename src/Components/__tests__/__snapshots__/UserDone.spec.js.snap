// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDone component Render Correctly 1`] = `
<section
  className="mainSection"
>
  <section
    className="formSection"
  >
    <section
      className="userImg"
    >
      <img
        alt=""
        src="user-check-solid.svg"
      />
    </section>
    <p
      className="verifyText"
    >
      Please verify your email address, you should have received an email from us already
    </p>
  </section>
</section>
`;
